File: 2.1SEM/Python/50sual FM (Olimpiada)/A46.py
n = int(input("eded: "))
result=0
count=1
while n>=1:
    if not n%2 == 0:
        result+=n%10*count
        count*=10
    n=n//10
else: print(result)

#dordreqemli ededin cut reqemlerini atir, yerde qalanlari chap edir

File: 2.1SEM/C++/50sual FM (Olimpiada)/A46.cpp
#include<iostream>
using namespace std;
int main(){
	int n,result=0,count=1;
	cin>>n;
	while(n>0){
		if(n%2!=0){
			result+=n%10*count;
			count*=10;
		}
		n/=10;
	}
	cout<<result;
return 0;
}

//dordreqemli ededin cut reqemlerini atir, yerde qalanlari chap edir
